### DISCUSSION TITLE
Welcome to homesecurity Discussions!

### DISCUSSION THREAD

**Discussion by dataplayer12** [dataplayer12/homesecurity#18] [General]
## 👋 Welcome!
  This repo has generated many insightful issues over time. This is a place to ask questions that would not be suitable as an issue. Please feel free to:
  * Ask questions.
  * Share ideas.
  * Engage.
  * Provide constructive criticism. It makes everyone better💪

**Comment by mlawtk** [comment]
Hi.  

I was wondering how the project is going and is this setup worth doing?  

Considering this at the moment.
